packages/suite/src/components/suite/Ticker/NoRatesTooltip.tsx

```diff
@@ -15,18 +15,12 @@ const NoRatesMessage = styled.div`
 interface NoRatesTooltipProps extends Partial<typeof Tooltip> {
     customText?: TranslationKey;
     customTooltip?: TranslationKey;
-    iconOnly?: boolean;
     className?: string;
 }
 
-export const NoRatesTooltip = ({
-    customText,
-    iconOnly,
-    customTooltip,
-    className,
-}: NoRatesTooltipProps) => (
+export const NoRatesTooltip = ({ customText, customTooltip, className }: NoRatesTooltipProps) => (
     <NoRatesMessage className={className}>
-        {!iconOnly && <Translation id={customText || 'TR_FIAT_RATES_NOT_AVAILABLE'} />}
+        <Translation id={customText || 'TR_FIAT_RATES_NOT_AVAILABLE'} />
         <TooltipSymbol
```

Comment:
nit icon color is weird
![Screenshot 2024-04-18 at 10 55 38](https://github.com/trezor/trezor-suite/assets/33235762/121fc56b-b385-45e0-a24f-4cfe570f466d)

Reply:
e4991bc3e28cd6efa3cd43aa10f8a7a829b736b8